pkg/validation/policy/validate.go

```diff
@@ -123,6 +123,10 @@ func Validate(policy, oldPolicy kyvernov1.PolicyInterface, client dclient.Interf
 	spec := policy.GetSpec()
 	background := spec.BackgroundProcessingEnabled()
 	mutateExistingOnPolicyUpdate := spec.GetMutateExistingOnPolicyUpdate()
+	if policy.GetSpec().CustomWebhookConfigurations() &&
+		!kubeutils.HigherThanKubernetesVersion(client.GetKubeClient().Discovery(), logging.GlobalLogger(), 1, 27, 0) {
```

Comment:
For 1.27, the feature flag `AdmissionWebhookMatchConditions` is unset by default. Is there a way to check this feature flag? 
I faced such an issue with VAPs and I used the discovery to do such a check but I am not sure if we can do the same for WehbookMatchConditions since it isn't a resource.

Reply:
There's no in-cluster way to check feature flags. There is a metric which can be scraped but that is probably too much to do.

Reply:
https://kubernetes.slack.com/archives/C0EG7JC6T/p1706112574262799?thread_ts=1706112365.454499&cid=C0EG7JC6T

Reply:
I'm clarifying this in the doc. I don't know what we can do on the code side as the feature is enabled via APIServer confgurations.